rasa/engine/storage/local_model_storage.py

```diff
@@ -26,6 +27,32 @@
 MODEL_ARCHIVE_METADATA_FILE = "metadata.json"
 
 
+@contextmanager
+def windows_safe_temporary_directory(
```

Comment:
from a code organisation standpoint, not sure if this should be in another utils file or stay here 🤔

Reply:
since it is only used in here, i don't see the need to move it somewhere else